extensions/eclipse-che-theia-git-provisioner/src/browser/git-preferences.ts

```diff
@@ -8,41 +8,48 @@
  * SPDX-License-Identifier: EPL-2.0
  ***********************************************************************/
 
+import {
+  PreferenceContribution,
+  PreferenceProxy,
+  PreferenceSchema,
+  PreferenceService,
+  createPreferenceProxy,
+} from '@theia/core/lib/browser';
+
 import { interfaces } from 'inversify';
-import { createPreferenceProxy, PreferenceProxy, PreferenceService, PreferenceContribution, PreferenceSchema } from '@theia/core/lib/browser';
 
 export const GitConfigSchema: PreferenceSchema = {
-    'type': 'object',
-    'properties': {
-        'git.user.name': {
-            'type': 'string',
-            'description': 'Your full name to be recorded in any newly created commits.',
-            'default': undefined
-        },
-        'git.user.email': {
-            'type': 'string',
-            'description': 'Your email address to be recorded in any newly created commits.',
-            'default': undefined
-        }
-    }
+  type: 'object',
+  properties: {
+    'git.user.name': {
+      type: 'string',
+      description: 'Your full name to be recorded in any newly created commits.',
+      default: undefined,
+    },
+    'git.user.email': {
+      type: 'string',
+      description: 'Your email address to be recorded in any newly created commits.',
+      default: undefined,
+    },
```

Comment:
Is it okay, that formatter adds extra commas here?

Reply:
yes, it's because if you add new properties, you don't want to have an extra diff line that adds a comma
this is why in 'modern' languages, these kind of extra commas are there

if you don't like them, we could patch default prettier, but then it means extra diff

Reply:
Let it be with an extra comma. It just a caught my attention during the review :)

Reply:
https://prettier.io/docs/en/options.html#trailing-commas
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Trailing_commas

Reply:
> https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Trailing_commas

Yep, this is useful feature, thanks for pointing it out 🙂